server/opts_test.go

```diff
@@ -761,6 +762,111 @@ func TestNewStyleAuthorizationConfig(t *testing.T) {
 	}
 }
 
+// Test new nkey users
+func TestNkeyUsersConfig(t *testing.T) {
+	confFileName := "nkeys.conf"
```

Comment:
You may want to use `createConfFile(t *testing.T, content []byte) string` which will use a temp file name/location. So it would look like:
```
  confFileName := createConfFile(t, []byte(`...`)
  defer os.Remove(confFileName)
  opts, err := ProcessConfigFile(...
```

Reply:
ok I can do that. Thanks.